samples/oauth_php/VetsApiClaims/app/Http/Controllers/ClaimsController.php

```diff
@@ -18,6 +18,51 @@ public function index(Request $request)
             $response = $client->request('GET', $url);
             $claims = json_decode($response->getBody()->getContents(), true);
             return view('claims', ['claims' => $claims['data']]);
-        } 
+        }
+    }
+
+    public function poa_form(Request $request)
+    {
+        return view('poa_form', []);
+    }
+
+    public function poa_form_show(Request $request, $id)
+    {
+        $oauth_session = OauthSession::find($request->session()->get('id'));
```

Comment:
may need try/catch on OauthSession - or some kind of short circuit if it fails/not found - I don't recall how Laravel handles these.

Reply:
there is a middleware AFAIK that checks for auth before routing. If I go to one of these routes without being authenticated it redirects home